index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+if (process.env.NEW_RELIC_ENABLED !== 'false') {
```

Comment:
- I strongly suggest `=== 'true'` so it won't be activated by default
- I would add a check for `process.env.NEW_RELIC_LICENSE_KEY` as well because requiring newrelic without a license key is a big memory-leaking time bomb (cf yesterday https://transcovo.atlassian.net/wiki/display/PD/Dishwasher+memory+leak)
If no license key I would personally not require NR and log an error.